catalyst_voices_packages/catalyst_cardano_serialization/lib/src/builders/transaction_builder.dart

```diff
@@ -19,7 +19,7 @@ final class TransactionBuilder extends Equatable {
   ///
   /// Enough [inputs] must be provided to be greater or equal
   /// the amount of [outputs] + [fee].
-  final List<TransactionUnspentOutput> inputs;
+  final Set<TransactionUnspentOutput> inputs;
```

Comment:
Please add this to breaking changes. Yes, it's very minor but we don't graduate breaking changes, if a change is breaking then it should be listed.